CONFIGURATION.md

```diff
@@ -114,6 +114,9 @@ The other placeholders are specified separately.
   # HTTP proxy server to use to connect to the targets.
   [ proxy_url: <string> ]
 
+  # Skip DNS resolution and URL change when an HTTP proxy is set.
```

Comment:
nit: making it explicit that we are talking about `proxy_url` config option here.

```suggestion
  # Skip DNS resolution and URL change when an HTTP proxy (proxy_url) is set.
```